FRAuth/FRAuth/Manager/KeychainManager.swift

```diff
@@ -226,6 +226,31 @@ struct KeychainManager {
         return nil
     }
     
+    //  MARK: - FROptions
+    
+    /// Returns current session's Token object that represents SSO Token
+    func getFROptions() -> Data? {
+        if let frOptionsData = self.privateStore.getData("FROptions") {
+            return frOptionsData
+        }
+        else {
+            return nil
+        }
+    }
+    
+    
+    /// Stores SSOToken into designated Keychain Service, or removes SSOToken when nil
+    /// - Parameter ssoToken: Token object
+    /// - Returns: Boolean result of operation
+    @discardableResult func setFROptions(frOptionsData: Data?) -> Bool {
+        if let frOptions = frOptionsData {
+            return self.privateStore.set(frOptions, key: "FROptions")
+        }
+        else {
+            return self.privateStore.delete("FROptions")
```

Comment:
I think we should start extracting string literals/constants into let constants. Here "FROptions" is used 3 times.